src/helpers/steg/forelder.ts

```diff
@@ -10,31 +10,39 @@ import { ESvar, ISpørsmål, ISvar } from '../../models/felles/spørsmålogsvar'
 import { harValgtSvar } from '../../utils/spørsmålogsvar';
 import { erDatoGyldigOgInnaforBegrensninger } from '../../components/dato/utils';
 import { DatoBegrensning } from '../../components/dato/Datovelger';
+import { erGyldigDato } from '../../utils/dato';
 
 export const erAlleForeldreUtfylt = (foreldre: IForelder[]) =>
   foreldre.every((forelder) => erForelderUtfylt(forelder));
 
 export const erForelderUtfylt = (forelder: IForelder): boolean | undefined => {
-  const { borINorge, land, avtaleOmDeltBosted } = forelder;
+  const { navn, fødselsdato, ident, borINorge, land, avtaleOmDeltBosted } = forelder;
   const utfyltBorINorge =
     borINorge?.verdi || (borINorge?.verdi === false && land?.verdi !== '');
 
+  const utfyltFødselsdato = (
+    fødselsdato?.verdi !== '' ? erGyldigDato(fødselsdato?.verdi) : true
+  );
```

Comment:
Mulig jeg misforstår, men burde det ikke være: 
```suggestion
  const utfyltFødselsdato = (
    fødselsdato?.verdi !== '' ? erGyldigDato(fødselsdato?.verdi) : false
  );
```
sånn jeg leser det vil tom verdi gi `utfyltFødselsdato = true`

Reply:
Det er riktig at en tom verdi skal gi true. Dette er fordi fødselsdato feltet er optiona. (fyll ut kun hvis du vet). Så vi lar dem gå videre enten hvis feltet er helt tomt eller kun hvis de har fylt ut en dato som er i et gyldig format. Kan forbedre navnet på konstanten slik at det gir litt mer mening. :)

Reply:
Det gir mening :) Mest beskrivende variabelnavn blir kanskje `fødselsdatoErIkkeUgyldig` 🙈

Reply:
Og kan vel også skrive: 
```
const utfyltFødselsdato = erGyldigDato(fødselsdato?.verdi) || fødselsdato?.verdi === ''
```
men vet ikke om det er lettere å forstå

Reply:
Endret navn i ny commit